lib/qstash/url_group.rb

```diff
@@ -9,12 +9,12 @@
 module QStash
   module URLGroup
     class << self
-      def add_endpoints(url_group_name, endpoints: [], headers: {})
-        AddEndpoints.call(url_group_name, endpoints: endpoints, headers: headers)
+      def add_endpoints(url_group_name:, endpoints: [], headers: {})
+        AddEndpoints.call(url_group_name: url_group_name, endpoints: endpoints, headers: headers)
       end
```

Comment:
Match the code with the Readme instructions

Reply:
This is a breaking change - since I don't know who is using the gem - better off updating README instead please.

Reply:
Yes, this is a breaking change that's why I'm proposing a new Version. 

According to [semver.org#4](https://semver.org/#spec-item-4) this gem is in development, The API is expected to change, so I changed only the patch level of the version number. 

I have no problem with changing the Readme, but seems more consistent with the rest of the API to use the keyword args.

Changing something in the main branch should not be a problem, as it is expected to change, as long as follow the semver rules at the moment you push a new version to rubygems.